lib/beaker/options/presets.rb

```diff
@@ -181,6 +181,7 @@ def presets
           :ec2_yaml               => 'config/image_templates/ec2.yaml',
           :help                   => false,
           :collect_perf_data      => 'none',
+          :aws_keyname_modifier   => '',
```

Comment:
Or default this to a random string, but once you have a random input, what requirements would bring about a non-random or predictable one?

Perhaps there is value in putting the job_name anyway, but that feels like a tangent to the regression.